xds/internal/testutils/e2e/clientresources.go

```diff
@@ -160,7 +158,7 @@ func DefaultServerListener(host string, port uint32, secLevel SecurityLevel) *v3
 		}
 	}
 	return &v3listenerpb.Listener{
-		Name: fmt.Sprintf(ServerListenerResourceNameTemplate, fmt.Sprintf("%s:%d", host, port)),
+		Name: name,
```

Comment:
Do you want to have the ability to name the resource separately?

If this is just to fix `[]`, use https://golang.org/pkg/net/#JoinHostPort?

Reply:
Done. Thanks.